assets/src/js/admin/reports/components/ProductRecommendation/index.tsx

```diff
@@ -3,8 +3,12 @@ import {__} from '@wordpress/i18n';
 import {createInterpolateElement} from '@wordpress/element';
 import {getWindowData} from '../../utils';
 import './style.scss';
+import {
+    RecommendedProductData,
+    useRecommendations,
+} from '../../../../../../../src/Promotions/sharedResources/hooks/useRecommendations';
 
-import {RecommendedProductData, useRecommendations} from '@givewp/promotions/sharedResources/hooks/useRecommendations';
+// import {RecommendedProductData, useRecommendations} from '@givewp/promotions/sharedResources/hooks/useRecommendations';
```

Comment:
If this is not used then it can be removed. If needed for historical reference then we can use the git history instead of commented code.

Reply:
Ah, I see that this is the relative vs namespaced path issue that you mentioned earlier.

Reply:
Yea Im having trouble with it. Im going to use the relative path. https://github.com/impress-org/givewp/pull/6802/commits/91e9b20986bfd8047af0731625385573de3f2fec

Reply:
@kjohnson Ok, I see now that the config shows the path including /sharedResources so including it in the import was incorrect. Its loads properly now with the alias https://github.com/impress-org/givewp/pull/6802/commits/7a7da2d0f017cf7f3eaf5ff8c44cb4c88df5b06e